package.json

```diff
@@ -1,10 +1,16 @@
 {
```

Comment:
- [ ] Please remove `package-lock.json` from the `.gitignore` file. This file ensures that any other user of your app installs the correct versions of the dependencies of your project.